types/global.d.ts

```diff
@@ -1,3 +1,15 @@
 declare module 'react-native-swipe-modal-up-down';
+type ModalType = import("./enums").ModalType
+type ModalScreen = import("./enums").ModalScreen
 
 type ValueOf<T> = T[keyof T];
+
```

Comment:
I understand this will make the interfaces "magically available" however, this is strongly discouraged as highlighted here
 https://www.typescriptlang.org/docs/handbook/declaration-files/templates/global-plugin-d-ts.html

Reply:
I will read more on this. Thanks